spyder_kernels/customize/spyderpdb.py

```diff
@@ -100,6 +100,8 @@ def __init__(self, completekey='tab', stdin=None, stdout=None,
         # has no effect in previous versions.
         self.report_skipped = False
 
+        self.executing = False
```

Comment:
Please add a comment on top of this variable to explain why it's necessary or how it's used.

Reply:
Slight change: What I am actually interested in is to know if a pdb prompt is waiting. I adjusted the code and moved `cmd_input` which should have been in `SpyderPdb`